osutil/disks/disks.go

```diff
@@ -147,6 +147,20 @@ type Partition struct {
 	KernelDevicePath string
 	// KernelDeviceNode is the kernel device node in /dev.
 	KernelDeviceNode string
+	// PartitionType is the type of structure, for example 0C in the case of a
+	// vfat partition on a DOS disk, or 0FC63DAF-8483-4772-8E79-3D69D8477DE4,
+	// which is ext4 on a GPT disk. This is always upper case.
+	PartitionType string
+	// FilesystemType is the type of filesystem i.e. ext4 or vfat, etc.
+	FilesystemType string
+	// SizeInBytes is the overall size of the partition/structure in bytes.
+	SizeInBytes uint64
```

Comment:
This is a super nitpick, but I'd move this field after the StartInBytes one.

Reply:
moved